libr/core/cmd_write.c

```diff
@@ -66,7 +66,7 @@ static bool encrypt_or_decrypt_block(RCore *core, const char *algo, const char *
 					int result_size = 0;
 					ut8 *result = r_crypto_get_output (cry, &result_size);
 					if (result) {
-						r_io_write_at (core->io, core->offset, result, result_size);
+						r_core_write_at (core, core->offset, result, result_size);
```

Comment:
Maybe i would go for `r_io_write_at` always opt in to the lowest API `r_core_write_at` calls `r_io_write_at`, so we can catch all writes. One example would be when we patch relocs in a relocatable object in ELF those changes wouldn't be visible since I use RIOBind and here is depending on `r_core_write_at`